drivers/coremidi/core_midi.cpp

```diff
@@ -51,21 +51,21 @@ Error MIDIDriverCoreMidi::open() {
 	OSStatus result = MIDIClientCreate(name, NULL, NULL, &client);
 	CFRelease(name);
 	if (result != noErr) {
-		ERR_PRINTS("MIDIClientCreate failed: " + String(GetMacOSStatusErrorString(result)));
+		ERR_PRINT(("MIDIClientCreate failed, code: " + itos(result)).utf8().get_data());
```

Comment:
I think you can still use `ERR_PRINTS` to avoid having to call `.utf8().get_data()` manually, no?